DocAI Incubator Tools/best-practices/Pre_Post_HITL_Bounding_Box_Mismatch.ipynb

```diff
@@ -0,0 +1,813 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #65.    def file_names(file_path):
</pre><p>Quite a few functions throughout these notebooks are reused, they could be moved into a utilities.py file to simplify the notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>